fitbenchmarking/fitbenchmarking
# Fixes #689: Pin iminuit<2

#### Description of Work
Pins version for iminuit until upgrade to version 2.0 can be done.

Fixes #689 

#### Testing Instructions

1. Install with optional extra: minuit
2. Verify version is not 2.x

Function: Does the change do what it's supposed to?

Tests: Does it pass? Is there adequate coverage for new code?

Style: Is the coding style consistent? Is anything overly confusing?

Documentation: Is there a suitable change to documentation for this change?
